docs/dev-tools/zombienet/tutorials/minimal-example.md

```diff
@@ -0,0 +1,229 @@
+            <th>Name</th>
+            <td>bob</td>
+        </tr>
+        <tr>
+            <th>Direct Link</th>
+            <td><a href="https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55312#explorer">https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55312#explorer</a></td>
+        </tr>
+        <tr>
+            <th>Prometheus Link</th>
+            <td><a href="http://127.0.0.1:55314/metrics">http://127.0.0.1:55314/metrics</a></td>
+        </tr>
+        <tr>
+            <th>Log Cmd</th>
+            <td>tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+            <tr>
+                <th colspan="3" align="center">
+                Node Information
+                </th>
+            </tr>
+        <tr>
+            <th>Name</th>
+            <td>collator01</td>
+        </tr>
+        <tr>
+            <th>Direct Link</th>
+            <td><a href="https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55316#explorer">https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55316#explorer</a></td>
+        </tr>
+        <tr>
+            <th>Prometheus Link</th>
+            <td><a href="http://127.0.0.1:55318/metrics">http://127.0.0.1:55318/metrics</a></td>
+        </tr>
+        <tr>
+            <th>Log Cmd</th>
+            <td>tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+        <tr>
+            <th>Parachain ID</th>
+            <td>100</td>
+        </tr>
+        <tr>
+            <th>ChainSpec Path</th>
+            <td>/var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2</td>
+        </tr>
+    </table>
+</div>
+
+!!! note 
+    If the IPs and ports are not explicitly defined in the configuration file, they may change each time the network is started, causing the links provided in the output to differ from the example.
+
+## Interacting with the spawned network
+
+### Connecting to the nodes
+
+After the network is launched, you can interact with it using the [Polkadot.js App](https://polkadot.js.org/apps/){target=_blank}. To do so, open your browser and use the provided links listed by the output as 'direct link'. For instance, in this particular case, as the ports may vary from spawning to spawning, to interact with the `alice` node, open [https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer](https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer){target=_blank} as it is the link provided in the output for the `alice` node. Moreover, you can also do this for the `bob` and `collator01` nodes.
+
+If you want to interact with the nodes more programmatically, you can also use the [Polkadot.js API](https://polkadot.js.org/api/){target=_blank}. For example, the following code snippet shows how to connect to the `alice` node using the Polkadot.js API and log some information about the chain and node:
+
+```typescript
```

Comment:
Marking this item as solved since it will be resolved when those folders are merged into the main branch. It will be addressed at https://opslayer.atlassian.net/browse/DRP-40

Reply:
@nhussein11 - when you move these to code snippets, please make sure you run prettier